precode.sh

```diff
@@ -6,38 +6,41 @@
 #   dir2
 #   dir3
 #       dir4
+mkdir -p /task/dir1 /task/dir2 /task/dir3/dir4
```

Comment:
Создаем от текущей директории, а не от корня файловой системы
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```